appveyor.yml

```diff
@@ -25,6 +24,17 @@ test:
     - '**\Sarif.Driver.UnitTests.dll'
     - '**\Sarif.FunctionalTests.dll'
     - '**\Sarif.ValidationTests.dll'
+    
+artifacts:
+  - path: 'bld\bin'
+
+notifications:
+  - provider: Email
+    to:
+      - vsarcplat@microsoft.com
+    subject: 'Sarif-SDK Build {{status}}'
+    message: 'In reference to {{commitID}}. View output <a href="{{buildURL}}">here</a> '
```

Comment:
It would be nice to add some combination of {{commitAuthorEmail}}, {{commitAuthor}}, {{commitDate}}, {{projectName}}, {{buildVersion}} too.

---
In reply to: [67393094](https://github.com/Microsoft/sarif-sdk/pull/364#discussion_r67393094) [](ancestors = 67393094)